apps/backoffice/src/components/api-keys/api-keys-groups/api-keys-groups.tsx

```diff
@@ -341,8 +341,8 @@ export const ApiKeysGroups = (props: ApiKeysGroupsProps) => {
         >
           <></>
         </LoaderSkeleton>
-        {mspData &&
-          mspData?.pagination.count > pagination.offset + pagination.limit && (
+        {Array.isArray(mspData?.value) &&
```

Comment:
Is the `isArray` check necessary?
TypeScript's typecheck already correctly models the value as an array of objects, and `useFetch` decodes the data using the codec specified in the fetch usage definition, in this case, `SubscriptionPagination`.
Here `mspData` can be _undefined_ or an object in the SubscriptionPagination format.
Therefore, I think it's enough to simplify with:
`{ mspData?.value.length === pagination.limit && (`
nothing else.